### PR TITLE
MODQM-306 Correction of actual and potential code smells

### PR DESCRIPTION
**Pull request by hasanepam**
## Purpose
_Omitting actual and potential code smells._

## Approach
_Created and used String constants to avoid repeated String values._

### CODE REVIEW

**Review by Shans-Kaluhin — APPROVED** [review]
Please add the title to the PR

File: src/main/java/org/folio/qm/validation/FieldValidationRule.java [hunk 1]
```diff
@@ -16,6 +16,8 @@
 public abstract class FieldValidationRule implements ValidationRule {
 
   public static final String CONTENT_COULDN_T_BE_EMPTY = "Content couldn't be empty";
```
Comment (psmagin) [inline src/main/java/org/folio/qm/validation/FieldValidationRule.java:18]:
```suggestion
  public static final String EMPTY_CONTENT_ERROR_MSG = "Content couldn't be empty";
```

File: src/main/java/org/folio/qm/validation/FieldValidationRule.java [hunk 2]
```diff
@@ -16,6 +16,8 @@
...
+  public static final String IS_UNIQUE_TAG = "Is unique tag";
```
Comment (psmagin) [inline src/main/java/org/folio/qm/validation/FieldValidationRule.java:19]:
```suggestion
  public static final String IS_UNIQUE_TAG_ERROR_MSG = "Is unique tag";
```

File: src/main/java/org/folio/qm/validation/FieldValidationRule.java [hunk 3]
```diff
@@ -16,6 +16,8 @@
...
+  public static final String IS_REQUIRED_TAG = "Is required tag";
```
Comment (psmagin) [inline src/main/java/org/folio/qm/validation/FieldValidationRule.java:20]:
```suggestion
  public static final String IS_REQUIRED_TAG_ERROR_MSG = "Is required tag";
```

**Review by psmagin — CHANGES_REQUESTED** [review]
Please make naming informative

**Pull request merged**